src/script/util/DraftStateUtil.ts

```diff
@@ -22,73 +22,52 @@ import {generateConversationInputStorageKey} from 'Util/util';
 import {MessageRepository} from '../conversation/MessageRepository';
 import {Conversation} from '../entity/Conversation';
 import {ContentMessage} from '../entity/message/ContentMessage';
-import {MentionEntity} from '../message/MentionEntity';
 import {StorageRepository} from '../storage';
 
-interface DraftMessage {
-  mentions: MentionEntity[];
-  reply?: ContentMessage;
-  replyEntityPromise?: Promise<ContentMessage>;
-  text: string;
-}
-
-type Reply = {
-  messageId?: string;
-};
-
-export interface Draft {
-  mentions: MentionEntity[];
-  reply: Reply;
-  text: string;
+export interface DraftState {
+  editorState: string | null;
+  messageReply?: Promise<ContentMessage>;
 }
 
 export const saveDraftState = async (
```

Comment:
Can you confirm that when loading this code with old draft (created with the previous version of the input bar), the draft will load in the input?

Reply:
Will actually not work. Need to discuss with product if it's okay if old drafts are not restored